lms/djangoapps/teams/search_indexes.py

```diff
@@ -0,0 +1,92 @@
+""" Search index used by haystack to load data into elasticsearch"""
+
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from search.search_engine_base import SearchEngine
+
+from .serializers import CourseTeamSerializer, CourseTeam
+
+
+class CourseTeamIndexer(object):
+    """
+    Index object for CourseTeam model instances
+    """
+    INDEX_NAME = "course_team_index"
+    DOCUMENT_TYPE_NAME = "course_team"
+    ENABLE_INDEXING_KEY = "ENABLE_TEAMS_INDEX"
+
+    def __init__(self, course_team):
+        self.course_team = course_team
+
+    def data(self):
+        """
+        Uses the CourseTeamSerializer to create a serialized course_team object
+        Adds in additional fields specific to search
+
+        Returns serialized object with additional search fields
+        """
+        serialized_course_team = CourseTeamSerializer(self.course_team).data
+        serialized_course_team['pk'] = self.course_team.pk
+        # add generally searchable content
+        serialized_course_team['content'] = {
+            'text': self.content_text()
+        }
+
+        return serialized_course_team
+
+    def content_text(self):
+        """
+        Generate the text field used for general search
+        """
+        return '{name} {description} {country} {language}'.format(
```

Comment:
Won't this allow some funky search matches? For instance, if you have this--

name description country language

Then you search for "description country", you would get a match for the whole phrase. But you shouldn't since description and country are in different fields.

I would have expected to see these individual fields added to the index, more like in /edx-notes-api/notesapi/v1/search_indexes.py.